docs/incidents/queries.md

```diff
@@ -221,20 +221,680 @@ These are a collection of endpoints that allow you to query incidents.
 
 ## Acknowledged Incidents
 
-<APIDetails :endpoint="$page.frontmatter.endpoints.acknowledged"/>
+<APIDetails :endpoint="$page.frontmatter.endpoints.acknowledged">
+
+::::: slot example
+
+:::: tabs :options="{ useUrlFragment: false }"
+
+::: tab "cURL"
+
+``` bash
+curl https://EXAMPLE.canary.tools/api/v1/incidents/acknowledged \
+  -d auth_token=EXAMPLE_AUTH_TOKEN \
```

Comment:
We should include `shrink=true` here as it's not required, but is suggested (it defaults to false for backwards compatibility)